File: JAVA Solutions/RemoveDuplicatesSortedArray80.java
public class RemoveDuplicatesSortedArray80 {
    public static void main(String[] args) {
        int[] nums = { 0, 0, 1, 1, 1, 1, 2, 3, 3 };
        int res = removeDuplicates(nums);
        System.out.println(res);
    }

    public static int removeDuplicates(int[] nums) {
        int i = 0;
        for (int it = 0; it < nums.length; it++) {
            if (i == 0 || i == 1 || nums[i - 2] != nums[it]) {
                nums[i] = nums[it];
                i++;
            }
        }
        return i;
    }
}

File: C++ Solutions/RemoveDuplicatesSortedArray80.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i = 0;
        for (int it = 0; it < nums.size(); it++) {
            if (i == 0 || i == 1 || nums[i - 2] != nums[it]) {
                nums[i] = nums[it];
                i++;
            }
        }
        return i;
    }
};
int main(){
    vector<int> nums={0,0,1,1,1,1,2,3,3};
    Solution s1;
    int res=s1.removeDuplicates(nums);
    cout<<res;
    return 0;
}